packages/neos-ui-backend-connector/src/Endpoints/index.ts

```diff
@@ -476,11 +476,16 @@ export default (routes: Routes) => {
                     throw new Error('.node-frontend-uri does not contain a valid href attribut');
                 }
 
-                const nodeContextPath = d.querySelector('.node-context-path');
-                if (!nodeContextPath) {
+                const nodeContextPathElement = d.querySelector('.node-context-path');
+                if (!nodeContextPathElement) {
                     throw new Error('.node-context-path is not found in the result');
                 }
 
+                const nodeContextPath = nodeContextPathElement.innerHTML.trim();
+                if (!nodeContextPath) {
+                    throw new Error('.node-context-path is empty');
+                }
```

Comment:
why did this work previously ??? 😂

Reply:
I suppose `.node-context-path` was never empty before, but occurred to be empty during the tests, because of all the backend changes.